Triangulation_2/doc/Triangulation_2/CGAL/draw_triangulation_2.h

```diff
@@ -3,41 +3,43 @@ namespace CGAL {
 /*!
 \ingroup PkgDrawTriangulation2
 
-opens a new window and draws a triangulation.  If the triangulation
-has constraints they are drawn.  If the face type has a member
-function `bool is_in_domain()` the faces inside and outside of the
-domain are drawn in different colors.
+opens a new window and draws a triangulation. Parameters of the drawing are taken from the optional graphics scene options parameter.
 
-A call to this function is blocking, that is the program continues as soon as the user closes the window. This function requires `CGAL_Qt5`, and is only available if the macro `CGAL_USE_BASIC_VIEWER` is defined.
+A call to this function blocks the execution of the program until the drawing window is closed. This function requires `CGAL_Qt5`, and is only available if the macro `CGAL_USE_BASIC_VIEWER` is defined.
 Linking with the cmake target `CGAL::CGAL_Basic_viewer` will link with
 `CGAL_Qt5` and add the definition `CGAL_USE_BASIC_VIEWER`.
 
-\tparam T2 a triangulation class derived from `Triangulation_2`  or `Constrained_triangulation_2`
+\tparam T2 a triangulation class derived from `Triangulation_2`
 \param at2 the triangulation to draw.
+\param gso the graphics scene options parameter, `Graphics_scene_options()` by default.
 
 */
 template<class T2>
-void draw(const T2& at2);
+void draw(const T2& at2,
+          const CGAL::Graphics_scene_options<T2,
+          typename T2::Vertex_handle,
+          typename T2::Finite_edges_iterator,
+          typename T2::Finite_faces_iterator>& gso=default);
 
```

Comment:
I do not understand why the signature in the documentation is lying. The actual signature is:
```c++
template <class Gt, class Tds, class GSOptions>
void draw(const CGAL::Triangulation_2<Gt, Tds>& at2, const GSOptions &gs_options,
```

With the current documentation, I was presuaded that it could not work if one includes two different `CGAL/draw_something.h>` headers.
`

Reply:
Isn't that also a problem for `CGAL::add_in_graphics_scene<POLY>()`   where `POLY`   looks like a template parameter.

Reply:
From the doc point of view, it is (maybe?) an implementation detail ? 

If we want to document exactly the type, the doc becomes hard to read.

Reply:
done